File: src/main/java/app/askresume/api/member/validator/PasswordValidator.java
package app.askresume.api.member.validator;

import app.askresume.global.error.ErrorCode;
import app.askresume.global.error.exception.BusinessException;

public class PasswordValidator {

    public static void passwordCheck(String password1, String password2) {
        if(!password1.equals(password2))
            throw new BusinessException(ErrorCode.SIGNUP_PASSWORD_INCONSISTENCY);
    }
}

File: src/main/kotlin/app/askresume/api/member/validator/PasswordValidator.kt
package app.askresume.api.member.validator

import app.askresume.global.error.ErrorCode
import app.askresume.global.error.exception.BusinessException

object PasswordValidator {
    fun passwordCheck(password1: String, password2: String) {
        if (password1 != password2)
            throw BusinessException(ErrorCode.SIGNUP_PASSWORD_INCONSISTENCY)
    }

}

